src/meta.js

```diff
@@ -12,7 +12,8 @@ const ALLOWED_ATTRS = [
     SDK_SETTINGS.ENABLE_3DS,
     SDK_SETTINGS.SDK_INTEGRATION_SOURCE,
     SDK_SETTINGS.CLIENT_METADATA_ID,
-    ATTRIBUTES.UID
+    ATTRIBUTES.UID,
+    SDK_SETTINGS.CSP_NONCE
```

Comment:
I was debugging this with @alan-bendjuya and learned that on second render the `data-csp-nonce` value was missing from the sdk script and it's needed so SPB can pass the nonce to the overlay component on button click.